.travis.yml

```diff
@@ -2,13 +2,19 @@ sudo: required
 matrix:
     include:
         - dist: bionic
+          env: _ros_dist=dashing
         - dist: focal
+          env: _ros_dist=foxy
+        - dist: focal
+          env: _ros_dist=galactic
+        - dist: focal
+          env: _ros_dist=rolling

```

Comment:
Please remove rolling from the travis's tests as there are no plans to distribute librealsense2 debian package for that release.

Reply:
I will remove it. Thanks!

Reply:
@doronhi done!